project/Dependencies.scala

```diff
@@ -3,16 +3,16 @@ import sbt._
 object Dependencies {
 
   object Versions {
-    val CatsEffect  = "0.8"
-    val Monix       = "3.0.0-M3"
-    val Fs2         = "0.10.2"
-    val Http4s      = "0.18.1"
-    val Tsec        = "0.0.1-M9"
-    val Circe       = "0.9.1"
-    val ScalaTest   = "3.0.4"
-    val ScalaCheck  = "1.13.4"
-    val Logback     = "1.2.1"
-    val TypesafeCfg = "1.3.1"
+    val CatsEffect  = "1.1.0"
+    val Monix       = "3.0.0-RC2"
+    val Fs2         = "1.0.2"
+    val Http4s      = "0.20.0-M4"
+    val Tsec        = "0.0.1-M11"
```

Comment:
I think `tsec` is quite abandoned and there was no release for `cats-effect-1.x`.

Reply:
removed